setup.py

```diff
@@ -16,15 +16,6 @@ def read(*parts):
         return fp.read()
 
 
-def find_version(*file_paths):
-    version_file = read(*file_paths)
-    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
-                              version_file, re.M)
-    if version_match:
-        return version_match.group(1)
-    raise RuntimeError("Unable to find version string.")
```

Comment:
I'd rather we keep the version in only one place.

Reply:
Yeah... I think so too now.

also, https://www.python.org/dev/peps/pep-0396/#specification (point 8)

Reply:
FWIW, I was trying to make setup.py as declarative as possible.